solr/solr-ref-guide/src/json-query-dsl.adoc

```diff
@@ -303,6 +303,30 @@ include::{example-source-dir}JsonRequestApiTest.java[tag=solrj-ipod-query-bool-c
 ====
 --
 
+Example of referencing <<json-query-dsl.adoc#additional-queries>>, <<json-query-dsl.adoc#tagging-in-json-query-dsl,tagging>> and <<other-parsers.adoc#boolean-query-parser,exclusions>>:
```

Comment:
The syntax for a link to a section that's on the same page is to simply use the section title, so:

- `<<json-query-dsl.adoc#additional-queries>>` should become `<<Additional Queries>>`
- `<<json-query-dsl.adoc#tagging-in-json-query-dsl,tagging>>` should become `<<Tagging in JSON Query DSL,tagging>>`

If you don't want the first one to be printed as "Additional Queries" (capitalized), you can add the link text as in the 2nd example